backend/views/dialog.ejs

```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Movie Pigeon</title>
+  </head>
+  <body>
+    <p>Hi <%= user.username %>!</p>
+    <p><b><%= client.name %></b> is requesting <b>full access</b> to your account.</p>
+    <p>Do you approve?</p>
+
+    <form action="/api/oauth2/authorize" method="post">
+      <input name="transaction_id" type="hidden" value="<%= transactionID %>">
+      <div>
+      <input type="submit" value="Allow" id="allow">
```

Comment:
Since we are using a REST api, maybe do not use the ejs webpage here instead, directly add a post to the router